utils/utils_test.go

```diff
@@ -57,6 +52,7 @@ func TestStringSliceDiff(t *testing.T) {
 }
 
 func TestGetIpAddress(t *testing.T) {
+	assert := assert.New(t)
```

Comment:
For consistency, we prefer to not use this approach, we pass the `t` parameter in each call.